prow.sh

```diff
@@ -767,6 +786,67 @@ install_csi_driver () {
         info "For container output see job artifacts."
         die "deploying the CSI driver with ${deploy_driver} failed"
     fi
+
+    # Call post-install hook(s) if defined
+    if [ -n "${CSI_PROW_DRIVER_POSTINSTALL}" ]; then
+        # Iterate through space-separated list of hook functions
+        for hook_func in ${CSI_PROW_DRIVER_POSTINSTALL}; do
```

Comment:
Why is this an array btw?

Reply:
I'm thinking about if we want to test all the storage beta features in one k/k job use array we could combine some hooks to avoid write a new one do all the configurations even if a new hook combine some hooks inside it. Is there something I missed?